README.md

```diff
@@ -16,11 +16,19 @@ go get github.com/stellar/golistcmp
 Usage of golistcmp:
   golistcmp <go list before> <go list after>
 
-Example:
+Example (built dependency graph comparison):
   git checkout master
-  go list -m -json all > go.list.master
+  go list -json -deps -test ./... | jq -s 'map(select(.Module != null) | .Module) | unique | .[]' > go.list.json.master
```

Comment:
Smart way to remove duplicates and get only the relevant data! 👏

Reply:
In Go 1.17 the -m option will produce these narrower results too. Looking forward to it arriving.